GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -76,6 +76,15 @@ private bool TryRunProductUpgrade()
             string error = null;
             Version newestVersion = null;
             bool isInstallable = false;
+
+            if (this.upgrader.IsNoneRing())
+            {
+                string message = "Upgrade ring set to None. No upgrade check was performed.";
+                this.tracer.RelatedInfo($"{nameof(this.TryRunProductUpgrade)}: {message}");

```

Comment:
`ReportInfoToConsole` (next line) will already write the message to the log file. Do we need this one as well? (I see the same applies in a few other places in this class too)

Reply:
right its there in other places too. Will clean this up in an upcoming PR.